### ISSUE TITLE
"Print as support" not working as expected?

### ISSUE THREAD

**Issue by EvillNooB** [Ultimaker/Cura#12653]
### Application Version

5.0.0

### Platform

Windows 10

### Printer

Ender 3

### Reproduction steps

1. Imported custom supports along with the model
2. Changed their printing mode to "print as support"
![image](https://user-images.githubusercontent.com/21310212/176968714-49df2e23-148f-4213-ac87-011395b837f8.png)

### Actual results

Shape gets replaced with ordinary support, so "print as support" works as a support blocker in reverse, marking the model parts to be supported, but not actually printing provided 3d model "as support"
![image](https://user-images.githubusercontent.com/21310212/176968807-c6662c96-bd4d-4c45-b414-35deb05e2e39.png)
![image](https://user-images.githubusercontent.com/21310212/176970185-f006c875-bdba-41ea-a083-270323618179.png)

### Expected results

Shapes should remain the same, but their settings should be replaced with support settings (infill, wall count, support interfaces and other unique settings) 
basically, same shape, but printed as if it was a support
![image](https://user-images.githubusercontent.com/21310212/176969185-52f078b5-a8b1-4a4d-a301-fc00b344d62f.png)

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

[test_part2.zip](https://github.com/Ultimaker/Cura/files/9031700/test_part2.zip)

**Comment by GregValiant** [comment]
It doesn't like that wedge shape for some reason.  Someone from the Cura Team will take a look.

**Comment by fieldOfView** [comment]
This is because of the setting "Drop Down Support Mesh", which defaults to true. In the per model settings tool, where you have set the wedges to "print as support", add the "Drop Down Support Mesh" and then turn it off. You are going to have to do that for each of the models.
